app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -93,6 +101,41 @@ class AddDeadlineActivity : AppCompatActivity() {
         textTime.text = selectedDate.toLocalTime().toString()
     }
 
+    /**
+     * Setup the PlacesAutocompleteClient used to select a location for a deadline
+     */
+    private fun initializePlacesAutocomplete() {
+        Places.initialize(applicationContext, getString(R.string.places_key))
+        // Must be commented for now
+        /*
+        Places.createClient(this)
+        */
+        initializeAutocompleteFragment()
+    }
+
+    /**
+     * Setup the AutocompleteSupportFragment with all its parameters
+     */
+    private fun initializeAutocompleteFragment() {
+        val autocompleteFragment =
+            supportFragmentManager.findFragmentById(R.id.autocomplete_fragment) as AutocompleteSupportFragment
+
+        autocompleteFragment
+            .setPlaceFields(listOf(Place.Field.ID, Place.Field.NAME)) //Returned fields
+            .setCountries(listOf("CH")) // Default startup country
+
+        autocompleteFragment.setOnPlaceSelectedListener(object : PlaceSelectionListener {
+            override fun onPlaceSelected(place: Place) {
+                // TO DO: Define here what we do with the returned place
```

Comment:
`TO DO` would better be in a single word, so that IDEs detect it

Reply:
Good point